x-pack/plugins/reporting/server/routes/generation.ts

```diff
@@ -45,10 +45,10 @@ export function registerJobGenerationRoutes(reporting: ReportingCore, logger: Lo
     }
 
     try {
-      const enqueueJob = enqueueJobFactory(reporting, logger);
+      const enqueueJob = enqueueJobFactory(reporting, logger.clone([exportTypeId]));
```

Comment:
We clone the logger and add the export type ID so that the logger received by the enqueue function is pre-decorated with the export type ID string. 

Doing it here is a cleanup, so that the various enqueue functions don't have to add the export type ID in separate places.

Reply:
A few changes have been made since this review comment:
8d98f1c042e
b3f01bfe4b0